Formula/o/openvino.rb

```diff
@@ -1,8 +1,9 @@
 class Openvino < Formula
   desc "Open Visual Inference And Optimization toolkit for AI inference"
   homepage "https://docs.openvino.ai"
-  url "https://github.com/openvinotoolkit/openvino/archive/refs/tags/2023.1.0.tar.gz"
-  sha256 "ff88596b342440185874ddbe22874b47ad7b923f14671921af760b15c98aacd6"
+  url "https://github.com/ilya-lavrenov/openvino/archive/5b462af28c701813d9a5991028fa398785e4beae.tar.gz"
```

Comment:
why not tag the commit? it failed with style check right now :)

Reply:
It's just a validation PR to check that we work correctly with new dependencies :) 
as far as I remember, tag was not required before..

Reply:
I would like to say rarely, only for some non github-based projects.